kratos/python/add_mesh_to_python.cpp

```diff
@@ -70,6 +60,11 @@ typedef MeshType::NodesContainerType NodesContainerType;
 typedef Geometry<Node<3> >::PointsArrayType NodesArrayType;
 typedef Geometry<Node<3> >::IntegrationPointsArrayType IntegrationPointsArrayType;
 
+array_1d<double,3> GetNormalFromCondition( Condition& dummy )
```

Comment:
It is defined, line 714:

~~~
    /**
     * It computes the normal of the geometry, if possible
     * @return The normal of the geometry
     */
    virtual array_1d<double, 3> Normal()
    {
        KRATOS_ERROR << "Calling base class Normal method instead of derived class one. Please check the definition of derived class." << *this << "Remember the normal can be computed just in geometries with a local dimension: "<< this->LocalSpaceDimension() << "smaller than the spatial dimension: " << this->WorkingSpaceDimension() << std::endl;
        return ZeroVector(3);
    }
~~~